deployment/cloud_composer_template.tf

```diff
@@ -102,11 +102,15 @@ resource "google_composer_environment" "example_environment" {
       airflow_config_overrides = {
         core-allowed_deserialization_classes = ".*"
       }
+      # Note: keep this in sync with .github/requirements.txt
       pypi_packages = {
         apache-airflow-providers-sendgrid = ""
-        jsonlines                         = ""
         fabric                            = ""
+        google-cloud-bigquery             = ""
+        google-cloud-storage              = ""
```

Comment:
I think we have google-cloud-bigquery & google-cloud-storage included?

https://cloud.google.com/composer/docs/concepts/versioning/composer-versions

Reply:
Got it, that's why they weren't listed here. We do need to include them in `.github/requirements.txt`, so do you think they're worth including here too? Without a version specified, I expect this to be a no-op.

Reply:
For some reason, Composer comes with the GPU version of TensorFlow by default, so `tensorflow-cpu` may cause issues. I'll comment these out and leave a note that these packages are pre-installed.